Better default limit value for query_accounts / query_transfers

I'm using Python client tigerbeetle==0.16.64

query_accounts / query_transfers do not return any rows unless `limit` field is explicitly set to a value greater than `0`. I guess it's because `limit` is 0 by default which is very counter-intuitive. I fixed our code but would be great to either change the default value to something (8189?) or document and/or make it a required parameter.

A simple change like this: https://github.com/aivarsk/tigerbeetle/pull/1/files will make linters complain
`error: Missing positional argument "limit" in call to "QueryFilter"  [call-arg]`

It says positional, but you can still pass limit as a kwarg anywhere in the argument list.

--- a/docs/coding/api-changes.md
+++ b/docs/coding/api-changes.md
@@ -569,6 +569,8 @@ The TigerBeetle Python Client `0.17.0` introduced the following breaking changes
   |Exception  |                                       |`ClientReleaseTooHighError`              |
   |Exception  |                                       |`TooMuchDataError`                       |
 
+- Removed all default initialization values from the types `AccountFilter` and `QueryFilter`.
+
 ### Example:
 
 Before:

--- a/src/clients/python/python_bindings.zig
+++ b/src/clients/python/python_bindings.zig
@@ -460,7 +460,12 @@ pub fn main() !void {
     inline for (mappings_state_machine) |type_mapping| {
         const ZigType, const python_name = type_mapping;
         const has_default_initialization = switch (ZigType) {
-            tb.CreateAccountResult, tb.CreateTransferResult, tb.AccountBalance => false,
+            tb.AccountFilter,
+            tb.QueryFilter,
+            tb.CreateAccountResult,
+            tb.CreateTransferResult,
+            tb.AccountBalance,
+            => false,
             else => true,
         };
 

--- a/src/clients/python/src/tigerbeetle/bindings.py
+++ b/src/clients/python/src/tigerbeetle/bindings.py
@@ -258,15 +258,15 @@ class CreateTransferResult:
 
 @dataclass
 class AccountFilter:
-    account_id: int = 0
-    user_data_128: int = 0
-    user_data_64: int = 0
-    user_data_32: int = 0
-    code: int = 0
-    timestamp_min: int = 0
-    timestamp_max: int = 0
-    limit: int = 0
-    flags: AccountFilterFlags = AccountFilterFlags.NONE
+    account_id: int
+    user_data_128: int
+    user_data_64: int
+    user_data_32: int
+    code: int
+    timestamp_min: int
+    timestamp_max: int
+    limit: int
+    flags: AccountFilterFlags
 
 
 @dataclass
@@ -280,15 +280,15 @@ class AccountBalance:
 
 @dataclass
 class QueryFilter:
-    user_data_128: int = 0
-    user_data_64: int = 0
-    user_data_32: int = 0
-    ledger: int = 0
-    code: int = 0
-    timestamp_min: int = 0
-    timestamp_max: int = 0
-    limit: int = 0
-    flags: QueryFilterFlags = QueryFilterFlags.NONE
+    user_data_128: int
+    user_data_64: int
+    user_data_32: int
+    ledger: int
+    code: int
+    timestamp_min: int
+    timestamp_max: int
+    limit: int
+    flags: QueryFilterFlags
 
 
 class CPacket(ctypes.Structure):

